styles/index.css

```diff
@@ -6,10 +6,18 @@
 }
 
 #session_page {
+  position: relative;
   width: 500px;
   margin: auto;
 }
 
+#statistics {
+  position: fixed;
```

Comment:
I recommend putting the statistics inside session_page, then `position: absolute`.